packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js

```diff
@@ -0,0 +1,82 @@
+import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
```

Comment:
This file should perhaps be called examination-timetable-submission-....
Register confuses with the other journey.

Also, should we be creating sub directories here to group the tests , the filenames could then be shorter and more specific.

Reply:
Renamed as examination-timetable-myself-submission